packages/expo-image-picker/src/ExponentImagePicker.web.ts

```diff
@@ -150,6 +157,7 @@ function readFile(targetFile: Blob): Promise<ImageInfo> {
             uri,
             width: image.naturalWidth ?? image.width,
             height: image.naturalHeight ?? image.height,
+            ...(options.base64 && { base64: uri.substr(uri.indexOf(',') + 1) }),
```

Comment:
From the `FileReader` [doc](https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsDataURL)

<img width="770" alt="Screenshot 2021-04-14 at 12 01 19" src="https://user-images.githubusercontent.com/11232797/114692703-237bef00-9d19-11eb-97bc-2a024b32f6cc.png">

Reply:
Could you add this note as a comment in the code?

Reply:
Sure, I added it.